Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -218,28 +236,59 @@ class SyncSettingsTableViewController: UITableViewController {
         return cell
     }
     
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+            case Sections.deviceList.rawValue:
+                return Strings.devices
+            case Sections.syncTypes.rawValue:
+                return Strings.Sync.syncSettingsTitle
+            default:
+                return nil
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, viewForFooterInSection sectionIndex: Int) -> UIView? {
+        switch sectionIndex {
+            case Sections.syncTypes.rawValue:
+                return configureInformationView(with: Strings.Sync.syncConfigurationInformationText)
+            default:
+                return nil
+        }
+    }
+
+    override func tableView(_ tableView: UITableView, heightForFooterInSection sectionIndex: Int) -> CGFloat {
+        switch sectionIndex {
+            case Sections.syncTypes.rawValue:
+                return UITableView.automaticDimension
```

Comment:
Does automatic dimension even work with footers this way?

Reply:
Oh yes they do work like this perfectly, didnt have any problem with this approach. We have to also define footer height zero so first section will not have extra spacing we dont want. But If you have a different approach you wanted I can change this.